drizzlepac/haputils/poller_utils.py

```diff
@@ -7,6 +7,7 @@
 """
 from collections import OrderedDict
 import os
+import pdb
```

Comment:
This appears to be a leftover from debugging and can be removed.

Reply:
issue resolved by commit [aca89a7](https://github.com/spacetelescope/drizzlepac/pull/911/commits/aca89a7d2ae66605db0e3e372742388e74991a3d)